src/main/java/teammates/common/datatransfer/InstructorAttributes.java

```diff
@@ -152,12 +146,9 @@ public InstructorAttributes(Instructor instructor) {
         
         if (instructor.getInstructorPrivilegesAsText() == null) {
             this.privileges = new InstructorPrivileges(Const.InstructorPermissionRoleNames.INSTRUCTOR_PERMISSION_ROLE_COOWNER);
-            this.instructorPrivilegesAsText = this.getTextFromInstructorPrivileges();
         } else {
-            this.instructorPrivilegesAsText = instructor.getInstructorPrivilegesAsText();
+            this.privileges = getInstructorPrivilegesFromText(instructor.getInstructorPrivilegesAsText());
```

Comment:
`this` is `InstructorAttributes` whereas `instructor` is an `Instructor` entity which does not have the variable `privileges`.